.travis.yml

```diff
@@ -48,6 +48,16 @@ language: python
 - &env-functional-shard_3
   <<: *env-functional
   PYTEST_ADDOPTS: "'-m shard_3_of_3'"
+- &py27-env-functional-shard_1
```

Comment:
This set of YAML anchors is not needed: those above hold exact copies of the same values as below.
Also, you call them "py27" while there's literally nothing pointing to python 2.7, they are as generic as ones without such prefix.

Reply:
I am really happy to find someone pickier than me \o/.
PYTEST_ADDOPTS: "'-m shard_3_of_3'", with single quotes, is functional wit python3, so i didn't change it. But it didn't work with python2
The attempt to remove the single quotes failed miserably PYTEST_ADDOPTS: "-m shard_3_of_3"
The current attempt to without the single quotes but with an = in place of the space appears to be functional PYTEST_ADDOPTS: "-m=shard_3_of_3".
Indeed, now, good question: is it better to keep an exception for py2, or is it better to, at least, try the "-m=shard_3_of_3" for all versions pf python?

Reply:
I'm fine with both, I just wanted to reduce duplication which leads to even more diverse semi-copypasted approaches in future